### DISCUSSION TITLE
Openssl installed, but not found in /opt/homebrew/{include,lib}

### DISCUSSION THREAD

**Discussion by smontanaro** [Homebrew/discussions#3850] [Everyday usage]
I'm trying to install libraries and include files for OpenSSL so I can compile Python from source (Python's GitHub source, not Homebrew's). `brew search` suggests it's installed (at least 1.1, that's the check mark, right?):

```
(python310) ~/src/python/cpython% brew search openssl
==> Formulae
glib-openssl                                              opensc
openssl@1.1 ✔                                             open-sp
openssl@3                                                 openfst
openslp                                                   opencsg
openssh                                                   openmsx
opensaml                                                  homebrew/portable-ruby/portable-openssl

==> Casks
openmsx                                opensc                                 opensim
```

but include and shared object files are in `/opt/homebrew/Cellar/openssl@1.1/1.1.1r`. Am I going to have to craft explicit library and include path values for `configure`/`make` for each package of interest (Tcl/Tk is next on my list), or is there some way to get  Homebrew to install symlinks in `/opt/homebrew/{lib,include}`? The poking around I did led me to nothing...

**Comment by SMillerDev** [comment]
For each package that is also provided by macOS you'll have to tell your system where to find them, yes. `/opt/Homebrew/opt/<name>` might be nicer for that though since it doesn't depend on a package version.

**Comment by smontanaro** [comment]
Thanks. I managed to build Python's _ssl module by configuring like so:

```
./configure --with-openssl=/opt/homebrew/opt/openssl@3
```

Whether I want OpenSSL 3 is another question, but for the moment I will go with it. I'm mostly interested in making sure as many of Python's unit tests run.
